src/Core/Manifest/VersionProvider.php

```diff
@@ -50,10 +50,10 @@ public function getVersion()
             return $version;
         }
         $modules = $this->getModules();
-        $lockModules = $this->getModuleVersionFromComposer(array_keys($modules ?? []));
+        $lockModules = $this->getModuleVersionFromComposer(array_keys($modules));
         $moduleVersions = [];
         foreach ($modules as $module => $title) {
-            if (!array_key_exists($module, $lockModules ?? [])) {
+            if (!array_key_exists($module, $lockModules)) {
```

Comment:
Have you validated that there's no way a non-string can appear here?

Reply:
yes, it comes from getModules that doesn't return null values
and from getModuleVersionFromComposer that doesn't return null values